src/Mod/TechDraw/Gui/QGIViewDimension.cpp

```diff
@@ -437,10 +442,9 @@ void QGIDatumLabel::setPosFromCenter(const double& xCenter, const double& yCente
 
     // Adjust for difference in tight and original bounding box sizes, note the y-coord down system
     QPointF tol_adj = m_tolTextOver->tightBoundingAdjust();
-    m_tolTextOver->justifyLeftAt(tolLeft + tol_adj.x(), middle - tol_adj.y(), false);
+    m_tolTextOver->justifyLeftAt(tolLeft + tol_adj.x(), middle + tol_adj.y()/2.0, false);
     tol_adj = m_tolTextUnder->tightBoundingAdjust();
-    m_tolTextUnder->justifyLeftAt(tolLeft + tol_adj.x(), middle + overBox.height() - tol_adj.y(),
-                                   false);
+    m_tolTextUnder->justifyLeftAt(tolLeft + tol_adj.x(), middle + overBox.height() + tol_adj.y()/2.0, false);
```

Comment:
I somewhat get the logic behind `tolLeft + tol_adj.x(), middle + overBox.height() + tol_adj.y()/2.0`. I am though considering if there is a simpler way. Perhaps this should be moved to a function.

Just thinking aloud here. Not that important for now.

Reply:
The logic is a little convoluted, but it's pretty unique to the over/under tolerance display, it also is pretty depended on the code above so I'm not really sure it makes sense to move it.